extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -55,32 +59,48 @@ function jetpack_mailchimp_block_load_assets( $attr ) {
 	}
 	$classes = implode( $classes, ' ' );
 
+	$button_styles = array();
+	if ( isset( $attr['customBackgroundButtonColor'] ) && strlen( $attr['customBackgroundButtonColor'] ) ) {
```

Comment:
How about only checking if the string is not empty, like so?

```suggestion
	if ( ! empty( $attr['customBackgroundButtonColor'] ) ) {
```

Unless I am missing something it would accomplish the same thing?

Reply:
Done in c5dbbb900ec2fc5eee01d4a1b215907a15d1cbe9